ciao-controller/command.go

```diff
@@ -86,6 +86,21 @@ func (c *controller) deleteInstance(instanceID string) error {
 	return nil
 }
 
+func (c *controller) assignFloatingIP(floatingIP payloads.FloatingIP) error {
+	// get node id.  If there is no node id we can't send a delete
+	i, err := c.ds.GetInstance(floatingIP.AssignFloatingIP.InstanceUUID)
+	if err != nil {
+		return err
+	}
+
+	if i.NodeID == "" {
+		return errors.New("Cannot delete instance not assigned to Node")
```

Comment:
yeah, it was a typo, that code section was modified, so it's not there anymore, thanks Manohar